cdk/cdk-resources/alert-configuration/src/index.ts

```diff
@@ -27,6 +22,13 @@ export interface CfnAlertConfigurationProps {
    */
   readonly eventTypeName?: CfnAlertConfigurationPropsEventTypeName;
 
+  /**
```

Comment:
Doesn't the alert configuration still have the readOnly issue?

Reply:
if the answer is yes, I believe this constructor will still need changes before releasing it

Reply:
yes. sure. But the changes should first flow through CFN. But Rn, it aligns with what is published.